chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/PendingChangeRequestEmailJob.java

```diff
@@ -0,0 +1,225 @@
+                for (List<String> rowValue : rows) {
+                    csvPrinter.printRecord(rowValue);
+                }
+            } catch (IOException e) {
+                LOGGER.error(e);
+            }
+        } catch (IOException e) {
+            LOGGER.error(e.getMessage(), e);
+        }
+        return temp;
+    }
+
+    private List<String> getHeaderRow(final List<CertificationBodyDTO> activeAcbs) {
+        List<String> row = new ArrayList<String>();
+        row.add("Developer Name");
+        row.add("Developer Contact Name");
+        row.add("Developer Contact Email");
+        row.add("Developer Contact Phone Number");
+        row.add("Change Request Type");
+        row.add("Change Request Status");
+        row.add("Change Request Comment");
+        row.add("Change Request Date");
+        row.add("Change Request Days Open");
+        for (CertificationBodyDTO acb : activeAcbs) {
+            row.add(acb.getName());
+        }
+        return row;
+    }
+
+    private List<List<String>> createChangeWebsiteRows(final List<CertificationBodyDTO> activeAcbs,
+            final Date currentDate)
+            throws EntityRetrievalException {
+        LOGGER.debug("Getting change website requests");
+        List<ChangeRequest> requests = changeRequestDAO.getAllPending().stream()
+                .collect(Collectors.<ChangeRequest>toList());
+        LOGGER.debug("Found " + requests.size() + "pending change requests");
+
+        List<List<String>> activityCsvRows = new ArrayList<List<String>>();
+        for (ChangeRequest changeRequest : requests) {
+            List<String> currRow = createEmptyRow(activeAcbs);
+            putChangeWebsiteActivityInRow(changeRequest, currRow, activeAcbs, currentDate);
+            activityCsvRows.add(currRow);
+        }
+        return activityCsvRows;
+    }
+
+    private void putChangeWebsiteActivityInRow(final ChangeRequest activity,
+            final List<String> currRow, final List<CertificationBodyDTO> activeAcbs, final Date currentDate) {
+        // Straightforward data
+        currRow.set(DEVELOPER_NAME, activity.getDeveloper().getName());
+        currRow.set(DEVELOPER_CONTACT_NAME, activity.getDeveloper().getContact().getFullName());
+        currRow.set(DEVELOPER_CONTACT_EMAIL, activity.getDeveloper().getContact().getEmail());
+        currRow.set(DEVELOPER_CONTACT_PHONE_NUMBER, activity.getDeveloper().getContact().getPhoneNumber());
+        currRow.set(CHANGE_REQUEST_TYPE, activity.getChangeRequestType().getName());
+        currRow.set(CHANGE_REQUEST_STATUS, activity.getCurrentStatus().getChangeRequestStatusType().getName());
+        currRow.set(CHANGE_REQUEST_COMMENT, activity.getCurrentStatus().getComment());
+        currRow.set(CHANGE_REQUEST_DATE, Util.getTimestampFormatter().format(activity.getCurrentStatus().getStatusChangeDate()));
+
+        // Calculated time open
+        Date changeRequestDate = activity.getCurrentStatus().getStatusChangeDate();
```

Comment:
Is the activity.getCurrentStatus().getStatusChangedDate() the date of the most recent status change? That's how I read it but maybe it's not. I would have thought you wanted to use the creation date of the change request here.